<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Accesories</class>
 <widget class="QWidget" name="Accesories">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sisyphus : Category Accesories</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icon/category_accessories.png</normaloff>../icon/category_accessories.png</iconset>
  </property>
  <widget class="QTableWidget" name="table_accesories">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>25</y>
     <width>950</width>
     <height>475</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>950</width>
     <height>500</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>110</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Category</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Available Version</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Installed Version</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Description</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="package_uninstall">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>530</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Uninstall package&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icon/package_uninstall.png</normaloff>../icon/package_uninstall.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="orphans_remove">
   <property name="geometry">
    <rect>
     <x>525</x>
     <y>530</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Remove orphan packages&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icon/remove_orphans.png</normaloff>../icon/remove_orphans.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="category_exit">
   <property name="geometry">
    <rect>
     <x>775</x>
     <y>530</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Close window&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icon/window_close.png</normaloff>../icon/window_close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="package_install">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>530</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Install package&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icon/package_install.png</normaloff>../icon/package_install.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>table_accesories</tabstop>
  <tabstop>package_install</tabstop>
  <tabstop>package_uninstall</tabstop>
  <tabstop>orphans_remove</tabstop>
  <tabstop>category_exit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
